src/psyclone/tests/dynamo0p3_builtins_test.py

```diff
@@ -23,180 +23,7 @@
                          "test_files", "dynamo0p3")
 
 # functions
-
-
-def test_dynbuiltin_missing_defs():
```

Comment:
Is there a reason for moving all of these tests to the end? If not, please could we have them back here as it makes the diff very unwieldy :-)

Reply:
Moved back majority of the tests where they were before. The simple built-ins tests (e.g. aX_plus_Y) are still ordered according to the order in documentation.